metrics/grafana/tikv_details.dashboard.py

```diff
@@ -5598,6 +5598,39 @@ def RocksDB() -> RowPanel:
                     ),
                 ],
             ),
+        ]
+    )
+    layout.row(
+        [
+            graph_panel(
+                title="Compactions running jobs",
```

Comment:
There is an existing `Compaction operations` panel for the "The count of compaction and flush operations". Would it make sense to report the "running jobs" metric in that panel as well?